src/client/index.js

```diff
@@ -1,13 +1,13 @@
-import React from 'react';
-import { render } from 'react-dom';
-import { Provider } from 'react-redux';
-import { AppContainer } from 'react-hot-loader';
+import React from "react";
```

Comment:
These should be single quotes.
Use single quotes in JS
Double quotes for element attributes e.g `<a href="...">`